### ISSUE TITLE
RuntimeError when loading the network to the GPU

### ISSUE THREAD

**Issue by cocokale** [openvinotoolkit/openvino#2476]
The loading works perfectly on CPU, however, when I try to load the network to the GPU plugin, it outputs a RuntimeError:

```
    exec_net = ie.load_network(network=net, device_name=args.device)
  File "ie_api.pyx", line 314, in openvino.inference_engine.ie_api.IECore.load_network
  File "ie_api.pyx", line 323, in openvino.inference_engine.ie_api.IECore.load_network
RuntimeError: Unsupported number of dimensions: 0
```
Has anyone encountered such an error before? Looking forward to some help, many thanks!

 

OpenVINO: 2020.4 Release

python: python 3.7

OS: Ubuntu 16.04 & Ubuntu 18.04

GPUs I've tried on:

Atom E3900 Series Integrated Graphics Controller

Intel Corporation HD Graphics 530

CPUs:

Intel(R) Atom(TM) Processor E3950 @ 1.60GHz

Intel(R) Core(TM) i7-6700T CPU @ 2.80GHz

**Comment by jgespino** [comment]
Hi @cocokale 

Could you provide more information (framework, topology, custom trained?) about the model you are using? Are you able to load other models to GPU? Would it be possible to share the model? 

Also, I noticed you posted the same question on the Intel Forums. I suggest only posting on one location to avoid duplicate effort. 

https://community.intel.com/t5/Intel-Distribution-of-OpenVINO/RuntimeError-when-loading-the-network-to-the-GPU/m-p/1213827#M20888

Regards,
Jesus

**Comment by jgespino** [comment]
Closing, please re-open if additional assistance is needed.

**Comment by cocokale** [comment]
Hi, sorry for the late reply.

I am able to load other models (custom trained) to the GPU plugin, which mainly consists of convolutional layers and fully connected layers.
The model I'm working on is also custom trained which mainly consists of bert layer (multi-head attention) and fully connected layers.

I'd like to share the model by sending an email to you with an OneDrive link. Is it OK for you?

BTW, could you please reopen this issue? Thanks!

**Comment by cocokale** [comment]
> Closing, please re-open if additional assistance is needed.

Hi, sorry for the late reply.

I am able to load other models (custom trained) to the GPU plugin, which mainly consists of convolutional layers and fully connected layers.
The model I'm working on is also custom trained which mainly consists of bert layer (multi-head attention) and fully connected layers.

I'd like to share the model by sending an email to you with an OneDrive link. Is it OK for you?

BTW, could you please reopen this issue? Thanks! @jgespino

**Comment by jgespino** [comment]
Hi @cocokale 

Yes, please share a link to your model via email. 

Regards,
Jesus

**Comment by jgespino** [comment]
Hi @cocokale 

Thanks for sending the model over, I will take a look and get back to you.

Regards,
Jesus

**Comment by vladimir-paramuzov** [comment]
@cocokale We had a fix recently for similar issue (#2453). Could you try the latest master and check if the error still reproduces?

**Comment by cocokale** [comment]
> @cocokale We had a fix recently for similar issue (#2453). Could you try the latest master and check if the error still reproduces?

Hi Vladimir, thanks for you help. The RuntimeError is solved after I build the latest master branch.

**Comment by jgespino** [comment]
@cocokale Did you re-open by mistake or do you need additional assistance?
